src/components/inventory/Ros.js

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from 'react';
+import { useRouteMatch } from 'react-router-dom';
+import { RegistryContext } from '../../store';
+import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import fallback from '../SpinnerFallback';
+
+const RosTab = () => {
+    const { params } = useRouteMatch('/:inventoryId');
+    const { getRegistry } = useContext(RegistryContext);
+
+    return <AsyncComponent
+        appName="ros"
+        module="./SystemDetail"
+        getRegistry={getRegistry}
+        fallback={fallback}
+        customItnlProvider
+        entity={ { id: params.inventoryId } }
```

Comment:
If you are going to use the solution with `useSelector` you can change the `entity` prop to `inventoryId={ params.inventoryId }`

Also no need for `customItnlProvider` as ROS is not using currently any intl service, right?

Reply:
Done!